README.md

```diff
@@ -53,3 +62,12 @@ Can be used to test the crash reporting feature of HockeyApp.
 hockeyapp.addMetaData(success:function, error:function, data:object):void
 ```
 Will be displayed in crash reports in HockeyApp.
+
+## CrossWalk considerations
+
+An application that is hosted in a CrossWalk View will show a blank screenshot unless the following requirements are met:
```

Comment:
This sentence could be clearer, that it refers to the screenshots created by `hockeyapp.feedbackModal`.